File: app/src/main/java/com/bth/running/storage/StorageManager.java
package com.bth.running.storage;

import com.bth.running.running.Run;

import java.util.List;

/**
 * @author Martin Macheiner
 *         Date: 05.09.2017.
 */

public interface StorageManager {

    List<Run> getRuns();

    void storeRun(Run run);

    void deleteRun(Run run);

}

File: app/src/main/java/at/shockbytes/corey/running/storage/StorageManager.kt
package at.shockbytes.corey.running.storage


import at.shockbytes.corey.running.running.Run

/**
 * @author Martin Macheiner
 * Date: 05.09.2017.
 */

interface StorageManager {

    val runs: List<Run>

    fun storeRun(run: Run)

    fun deleteRun(run: Run)

}
